[2024-05-29T14:53:03.840+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available scheduled__2024-05-28T00:00:00+00:00 [queued]>
[2024-05-29T14:53:04.178+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available scheduled__2024-05-28T00:00:00+00:00 [queued]>
[2024-05-29T14:53:04.180+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T14:53:04.197+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-29T14:53:04.208+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T14:53:04.407+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-05-28 00:00:00+00:00
[2024-05-29T14:53:04.433+0000] {standard_task_runner.py:55} INFO - Started process 528 to run task
[2024-05-29T14:53:04.509+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1_dag', 'is_api_available', 'scheduled__2024-05-28T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/1_dag.py', '--cfg-path', '/tmp/tmp170k8s8o']
[2024-05-29T14:53:04.534+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask is_api_available
[2024-05-29T14:53:05.464+0000] {task_command.py:376} INFO - Running <TaskInstance: 1_dag.is_api_available scheduled__2024-05-28T00:00:00+00:00 [running]> on host 4b56679d13b8
[2024-05-29T14:53:07.879+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-05-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-05-28T00:00:00+00:00
[2024-05-29T14:53:07.889+0000] {http.py:120} INFO - Poking: api/
[2024-05-29T14:53:08.441+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-05-29T14:53:08.850+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-05-29T14:53:10.253+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-05-29T14:53:10.651+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=1_dag, task_id=is_api_available, execution_date=20240528T000000, start_date=20240529T145303, end_date=20240529T145310
[2024-05-29T14:53:10.836+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-29T14:53:10.969+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
